src/Symfony/Component/Finder/Finder.php

```diff
@@ -136,13 +136,89 @@ public function depth($level)
      *
      * @return $this
      *
+     * @deprecated since 4.1, to be removed in 4.2
      * @see strtotime
      * @see DateRangeFilterIterator
      * @see DateComparator
      */
     public function date($date)
     {
-        $this->dates[] = new Comparator\DateComparator($date);
+        $this->dateModified($date);
+
+        return $this;
+    }
+
+    /**
+     * Adds tests for file dates (last accessed).
+     *
+     * The date must be something that strtotime() is able to parse:
+     *
+     *   $finder->date('since yesterday');
```

Comment:
The examples in this PHPdoc should be updated to replace `->date(...` with the new methods.